main.py

```diff
@@ -112,7 +111,6 @@ def class_trainer():
         transform=transforms.ToTensor()),
         batch_size=config.batch_size, shuffle=True, drop_last=True)
 
-        
     test_ds = DataLoader(
```

Comment:
is it possible to move the DataLoader's into data_utils.py?

Reply:
Yes, will fix.